scaaml/plot.py

```diff
@@ -70,13 +70,13 @@ def plot_comparaison(traces, labels, title):
     for idx, trace in enumerate(traces):
         label = labels[idx]
         if label == "SNR":
-            color = '#8E24AA'
-        elif 'activation' in label.lower():
-            color = '#64DD17'
-        elif 'grad' in label.lower():
-            color = '#26A69A'
-        elif 'scald' in label.lower():
-            color = '#03A9F4'
+            color = "#8E24AA"
+        elif "activation" in label.lower():
+            color = "#64DD17"
```

Comment:
See PR #66  for comments on these.
Try to avoid including the same fixes in 2 distinct PRs. Otherwise I review the same changes twice :)